thoth/slo_reporter/sli_kebechet.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# slo-reporter
+# Copyright(C) 2020 Sai Sankar Gochhayat
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This file contains class for Kebechet."""
+
+import logging
+import os
+
+from typing import Dict, List, Any
+
+from .sli_base import SLIBase
+from .sli_template import HTMLTemplates
+from .configuration import Configuration
+
+_INSTANCE = "dry_run"
+_INTERVAL = "7d"
```

Comment:
could you use `Configuration._INTERVAL`? so we reuse same value across all templates.